### DISCUSSION TITLE
Monolithic Structure

### DISCUSSION THREAD

**Discussion by diksham1** [lanukahjo/Image-Splicing-Localisation#3] [General]
We'll need to keep monolithic structure since colab does not work with multiple files easily. Loading entire repo every single time is just not feasible. Since it's experimental, I guess monolithic should be fine

**Comment by lanukahjo** [comment]
Okay. That should not be a problem. We will make it less monolithic from
within monolithic (pseudo-monolothic). Just we will decide the structure.
And let us use 4 space tabs instead of 2 space tabs.

**Comment by meganindya** [comment]
Why do you need colab?

**Comment by lanukahjo** [comment]
Because Diksha's local machine is Pentium.

**Comment by meganindya** [comment]
for running `.ipynb` files?

**Comment by meganindya** [comment]
@lanukahjo enable `discussions` in settings.
